src/main/scala/akka/persistence/redis/RedisUtils.scala

```diff
@@ -18,40 +18,61 @@ package persistence
 package redis
 
 import actor._
-
 import _root_.redis._
-
+import akka.persistence.utils.HostAndPort
 import com.typesafe.config.Config
 
 import scala.collection.JavaConverters._
 
 object RedisUtils {
 
-  def host(conf: Config) = conf.getString("redis.host")
+  def host(conf: Config): String = conf.getString("redis.host")
+
+  def port(conf: Config): Int = conf.getInt("redis.port")
 
-  def port(conf: Config) = conf.getInt("redis.port")
+  def database(conf: Config): Option[Int] =
+    if (conf.hasPath("redis.database")) Some(conf.getInt("redis.database"))
+    else None
 
-  def database(conf: Config) = if (conf.hasPath("redis.database")) Some(conf.getInt("redis.database")) else None
+  def password(conf: Config): Option[String] =
+    if (conf.hasPath("redis.password")) Some(conf.getString("redis.password"))
+    else None
```

Comment:
If you put the `if`statement on several lines, please also add a line break after the condition and the `else` keyword.